### ISSUE TITLE
Jira bot edits mangle autolinked GitHub references

### ISSUE THREAD

**Issue by onnodb** [integrations/jira#141]
To reproduce: create a Pull Request that references _both_ a Jira issue, as well as an autolinked reference to another PR. For example, if I create a PR with the following body:
```
This PR addresses [JIRA-001]. See also my_org/my_project#123
```
Then I end up with a PR body in which the Jira issue is correctly linked, but the reference to PR 123 in "my_project" is dropped (i.e., shows up as plain text, not a link).

If I would have created the PR body without the Jira issue link, the autolinked reference would have been just fine.

**Comment by tcbyrd** [comment]
@onnodb Thanks for reporting this. I think I know what's causing that and will work on a fix soon.

**Comment by onnodb** [comment]
Great, thanks a lot!

**Comment by tcbyrd** [comment]
@onnodb check out #161 and let me know if you think about that proposed solution.

**Comment by onnodb** [comment]
That does look like an excellent solution!

**Comment by tcbyrd** [comment]
Fixed by #165!
